__TESTS__/unit/fromJson/transcode.fromJson.test.ts

```diff
@@ -0,0 +1,31 @@
+import {fromJson} from "../../../src/internal/fromJson";
+
+describe('transcode.fromJson', () => {
+  it('should generate a url with transcode actions from array of models', function () {
+    const transformation = fromJson({actions:[
+      { actionType: 'keyframeInterval', interval: 20},
+      { actionType: 'fps', fps: 6 },
+      { actionType: 'fps', fps: {from: 6, to: 8} },
+      { actionType: 'fps', fps: {from: 6} },
+      {actionType: 'bitRate', bitRate: 6},
+      {actionType: 'bitRate', bitRate: '500k', constant: true},
+      {actionType: 'audioCodec', audioCodec: 'aac'},
+      {actionType: 'audioFrequency', audioFrequencyType: 'freq8000'},
+      {actionType: 'streamingProfile', profile: 'fullHd'},
+      {actionType: 'toAnimated', animatedFormat: 'gif', delay: 20, sampling: '4s'}
+    ]});
+
```

Comment:
Can we combine all tests to this one file? 

It seems a bit simpler to test fromJSON() and then toJSON(), since the output of the latter is the input of the first